samples/http-cloudevents/client.py

```diff
@@ -21,44 +21,38 @@
 
 
 def send_binary_cloud_event(url):
-    # define cloudevents data
+    # This data defines a binary cloudevent
     attributes = {
-        "id": "binary-event-id",
-        "source": "localhost",
-        "type": "template.http.binary",
-        "dataontenttype": "application/json",
-        # Time will be filled in automatically if not set
-        "time": "2018-10-23T12:28:23.3464579Z",
+        "Content-Type": "application/json",
```

Comment:
Is ` "Content-Type": "application/json",` needed?
If so, we should eventually remove it as it should be automatically added as a HTTP header when constructing the request in `event.to_http`.

Reply:
removed it